sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -2098,6 +2098,192 @@ public virtual async Task<AnalyzeHealthcareEntitiesOperation> StartAnalyzeHealth
 
         #endregion
 
+        #region Dynamic Classify
+
+        /// <summary>
+        /// Performs dynamic classification on the given document.
+        /// For document length limits, maximum batch size, and supported text encoding, see
+        /// <see href="https://aka.ms/azsdk/textanalytics/data-limits"/>.
+        /// </summary>
+        /// <remarks>
+        /// This method is only available for <see cref="TextAnalyticsClientOptions.ServiceVersion.V2022_10_01_Preview"/>, and newer.
+        /// </remarks>
+        /// <param name="document">The document to analyze.</param>
+        /// <param name="language">The language that the documents are written in.</param>
+        /// <param name="options">The additional <see cref="DynamicClassifyOptions"/> used to configure the operation.</param>
+        /// <param name="cancellationToken">The <see cref="CancellationToken"/> controlling the lifetime of the request.</param>
+        /// <returns>
+        /// The collection of categories used to classify each document that was successfully analyzed.
+        /// </returns>
+        /// <exception cref="NotSupportedException">This method is only supported in service API version 2022-10-01-preview and newer.</exception>
+        /// <exception cref="RequestFailedException">The service returned a non-success status code.</exception>
+        /// <exception cref="ArgumentException"><paramref name="document"/> is an empty string.</exception>
+        /// <exception cref="ArgumentNullException">
+        /// <para><paramref name="document"/> is null.</para>
+        /// <para><paramref name="options"/> is null.</para>
+        /// </exception>
```

Comment:
Nit: 2 paragraphs just to document this way seems unnecessary and long. This won't grow. It's more typical just to say something like:

```c#
/// <exception cref="ArgumentNullException"><paramref name="document"/> or <paramref name="options"/> is null.</exception>
```

That also jives with official .NET documentation.